config/initializers/sufia.rb

```diff
@@ -208,7 +208,8 @@
      "2014-06" => "2014-06",
      "2014-11" => "2014-11",
      "2015-06" => "2015-06",
-     "2015-11" => "2015-11"
+     "2015-11" => "2015-11",
+     "2016-06" => "2016-06"
```

Comment:
@pgwillia yes, this is not related to handle update, or migration. This is for the edit form to have the value presented (which is only available for admin). Maybe we want to revisit how the values are populated on the edit form.